package.json

```diff
@@ -22,7 +22,7 @@
     "@types/jasmine": "^2.5.47",
     "browserify": "^14.4.0",
     "jasmine": "^2.6.0",
-    "lerna": "^2.0.0-rc.5",
+    "lerna": "^3.16.4",
```

Comment:
I had some problems publishing using recent versions, so I left it `rc.5`. Maybe the are fixed, but I'd prefer to update it myself; and maybe even switch to yarn-native multipackage support.

Reply:
The reason I switched to this is that I was having difficulties getting the dependencies to find each other when I was doing `yarn build` and this was the only solution I found for it.